Global/macOS.gitignore

```diff
@@ -3,7 +3,7 @@
 .LSOverride
 
 # Icon must end with two \r
-Icon
+# Icon

```

Comment:
Rather than comment this rule out, you should remove this whole section (including the comment above).